modules/videoio/src/cap_msmf.cpp

```diff
@@ -246,7 +299,7 @@ struct MediaType
         return wdiff + hdiff;
     }
     // check if 'this' is better than 'other' comparing to reference
-    bool isBetterThan(const MediaType& other, const MediaType& ref) const
+    bool VideoIsBetterThan(const MediaType& other, const MediaType& ref) const
```

Comment:
Usually video quality is compared not only by resolution, but also by bitrate and FRC mode